UnifiedApi/Worker/Grid/GridWorker.cs

```diff
@@ -183,6 +183,11 @@ public byte[] Execute(ITaskHandler taskHandler)
     {
       throw new WorkerApiException(e);
     }
+    catch (TargetInvocationException e)
+    {
+      throw e.InnerException ?? new Exception("Unknown exception in worker code",
```

Comment:
throw e ?

Reply:
e.InnerException would allow to see the actual error. Just throwing e would only show the "TargetInvocationException"